test/test-pages/pixnet/expected.html

```diff
@@ -184,5 +184,10 @@
         <p><span><span><span>內文有不定期的更新旅遊、露營圖文訊息 </span></span><span><span>謝謝!</span></span>
             </span>
         </p>
+        <div>
+            <p><img src="http://panel.pixfs.net/images/icons/tag_blue.gif" original="http://panel.pixfs.net/images/icons/tag_blue.gif" />
+                <a href="http://tags.pixnet.net/blog/user/stevenhgm/%E7%8E%89%E5%B3%B0%E9%81%93%E8%B7%AF" rel="tag" target="_blank">玉峰道路</a>, <a href="http://tags.pixnet.net/blog/user/stevenhgm/%E4%B8%8A%E5%AE%87%E8%80%81%E8%A7%80%E6%99%AF%E5%8F%B0" rel="tag" target="_blank">上宇老觀景台</a>, <a href="http://tags.pixnet.net/blog/user/stevenhgm/%E7%BE%8E%E6%A8%B9%E7%87%9F%E5%9C%B0" rel="tag" target="_blank">美樹營地</a>, <a href="http://tags.pixnet.net/blog/user/stevenhgm/%E6%8E%A7%E6%BA%AA%E5%90%8A%E6%A9%8B" rel="tag" target="_blank">控溪吊橋</a>, <a href="http://tags.pixnet.net/blog/user/stevenhgm/%E7%9F%B3%E7%A3%8A%E9%83%A8%E8%90%BD" rel="tag" target="_blank">石磊部落</a>
```

Comment:
Is this helpful? It's hard for me to be sure as the image is gone and I do not speak the language... but it looks like a series of links, which we normally omit in reader mode because they are not usually part of the core article (there's logic elsewhere that evaluates link density, ie links vs. non-link paragraph text and so on).